src/components/Activity/TimeTag.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GU, textStyle, useTheme } from '@aragon/ui'
+// import { formatDistance, formatDistanceStrict } from 'date-fns'
```

Comment:
are we planning to import those functions from `dayjs` now?

Reply:
Yes! https://github.com/aragon/court-dashboard/pull/100/files#diff-3f247f7e3ba0dcef7287f99eda17ace6R11-R12